README.md

```diff
@@ -13,6 +13,7 @@ Really! We needed to create it for internal use within [Apiary](https://apiary.i
 You ended up here for some reason though:
 
 - Looking for [API Blueprint spec](https://github.com/apiaryio/api-blueprint/)?
+- Looking for [Refract spec](https://github.com/refractproject/refract-spec)
```

Comment:
I think it will be better to link to http://api-elements.readthedocs.io/en/latest/ and use the current name "API Elements".